authentication/src/main/java/io/camunda/authentication/ConditionalOnInternalUserManagement.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.authentication;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+
+@Target({TYPE, METHOD})
+@Retention(RUNTIME)
+@Documented
+@ConditionalOnExpression("'${camunda.security.authentication.method:}'.toLowerCase() != 'oidc'")
+public @interface ConditionalOnInternalUserManagement {}
```

Comment:
Is this annotation required? Why @ConditionalOnAuthenticationMethod(AuthenticationMethod.OIDC) can't be used instead?